runner/operator/argos_legacy/0_1_0/bin/retrieve_samples_by_query.py

```diff
@@ -279,24 +279,24 @@ def build_dmp_sample(dmp_bam, patient_id, bait_set, tumor_type):
     sample['file_type'] = dmp_bam.file.file_type
     metadata = init_metadata()
     metadata[settings.SAMPLE_ID_METADATA_KEY] = sample_name
-    metadata['sampleName'] = format_sample_name(sample_name, specimen_type)
-    metadata['cmoSampleName'] = metadata['sampleName']
+    metadata[settings.CMO_SAMPLE_NAME_METADATA_KEY] = format_sample_name(sample_name, specimen_type)
```

Comment:
Can we confirm this is the right way to set the metadata for dmp samples. I think it needs the sampleName set in ciTag as well?